app/views/new_threads/show.html.erb

```diff
@@ -1,89 +1,79 @@
-       
-          </td>
-          <td>
-            <div class="like">
-              <%= button_to 'Flag', new_thread_flags_path(:new_thread_id => @new_thread), remote: true, :class => "like-bu flag-icon"%>
-            </div>
-          </td>
-          <td>
-            <span itemprop="answerCount"><%= @new_thread.replies.size %></span> &nbsp; Replies 
-          </td>
-        <tr>
-      </table>
-     
+       				<%end%>
+        			<% end %>
+
+
 
-      <span class="user-thread" itemprop="author" itemscope itemtype="http://schema.org/Person"><span itemprop="name"><%= @new_thread.user.username %></span></span>
-    </div>
-  </div>
-  <br><br><br><br><br>
-</div>
+		              <div class="thread-icons">
+		                  <%= button_to(like_counts_path(:new_thread_id => @new_thread), remote: true, :class => "like-bu") do %>
+		                    <span class="glyphicon glyphicon-thumbs-up" aria-hidden="true"></span>
+		                    <%= render :partial => "like_counts/new_thread", :locals => {:new_thread_id => @new_thread.id} %>
+		                  <% end %>
+		              </div>
+		              <div class="thread-icons">
+		                <%= button_to(dislike_counts_path(:new_thread_id => @new_thread), remote: true, :class=> "like-bu dis") do %>
+		                <span class="glyphicon glyphicon-thumbs-down" aria-hidden="true"></span>
+		                <%= render :partial => "dislike_counts/new_thread_dislike", :locals => {:new_thread_id => @new_thread.id} %>
+		                <% end %>
+		              </div>
+            		</div>
 
+				</div>
+			</div>
 
 
 
+			<h2 class="thread-header">Replies:</h2>
+				<% if @new_thread.replies.size == 0 %>
+				  <p class="no-reply">No one has replied to this question. Be the first to reply!</p>
+				<%end%>
+				<div id="replies">
+				  <%= render :partial => @new_thread.replies%>
+				</div>
 
 
+				<div class="col-xs-12 col-md-12 col-lg-12 thread-col">
+				<div class="thread-card">
 
+				<%= form_for([@new_thread, Reply.new], :html => {:class => 'reply-form'}, remote: true)  do |f| %>
+	<%= f.text_area :content, :class => 'form-control', :rows=>'3' %>
+  <% if user_signed_in? %>
+  <%= f.hidden_field :user_id, :value => current_user.id %>
+  <%end%>
+ <div id="reply-btn"><%= f.submit "Add Reply", :class => "reply-but btn btn-primary"%></div>
+ <% end %>
+ 					</div>
+ 					</div>
 
-<h2 class="thread-header">Replies:</h2>
-<% if @new_thread.replies.size == 0 %>
-  <p class="no-reply">No one has replied to this question. Be the first to reply!</p>
-<%end%>
-<div id="replies">
-  <%= render :partial => @new_thread.replies%>
-  
-</div>
-<%= form_for([@new_thread, Reply.new], remote: true ) do |f| %>
-  <p>  <%= f.text_area :content %>
-    <% if user_signed_in? %>
-    <%= f.hidden_field :user_id, :value => current_user.id %>
-    <%end%>
-  </p>
-  <p><%= f.submit "Add Reply", :class => "reply-but"%></p>
-<% end %>
+		</div>
+	</div>
+</div>
```

Comment:
That red symbol is there because you didn't leave a blank line at end of file. You don't need to edit commits for this, just next time when you edit these files, make sure that you leave a blank line.